package.json

```diff
@@ -34,6 +34,9 @@
     "lerna": "^4.0.0"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/material": "^5.10.15",
```

Comment:
You included a couple of full ui libraries. Even though it provides a convenient set of components but it also adds quite a bit of weight to the project and there are a couple of alternatives to it. I think we need to agree as a team whether or not we really want to use this (or any other) UI library for the frontend, because it's quite a big decision to make.

If I'm not mistaken, we already had a discussion about using some sort of UI library in this project, however I cannot recall what was the agreement, so, please, raise this question in the slack channel and we'll agree on the decision.